lib/shared/relative-module-resolver.js

```diff
@@ -23,8 +23,7 @@ const Module = require("module");
  * `Module.createRequire` is added in v12.2.0. It supports URL as well.
  * We only support the case where the argument is a filepath, not a URL.
  */
```

Comment:
It seems this whole comment can go away now. Unfortunately the diff doesn't extend far enough to let me leave a suggestion inline to make it easier. Do we use `Module` anywhere else in here? If not, `const { createRequire } = require("module");` could work too.

Reply:
well, the file has been moved to `@eslint/eslintrc`. We can just remove it after https://github.com/eslint/eslint/pull/14608 get merged.